applications/asset_tracker/src/main.c

```diff
@@ -1447,24 +1455,31 @@ void cloud_event_handler(const struct cloud_backend *const backend,
 			break;
 		}
 
-#if defined(CONFIG_AGPS)
-		err = gps_process_agps_data(evt->data.msg.buf,
-					    evt->data.msg.len);
+#if defined(CONFIG_NRF_CLOUD_CELL_POS)
+		double lat, lon;
+
+		err = nrf_cloud_cell_pos_process(evt->data.msg.buf, evt->data.msg.len);
```

Comment:
The code doesn't currently compile because this call has wrong number of arguments.